src/bun.js/bindings/BunObject.cpp

```diff
@@ -188,7 +197,19 @@ JSC_DEFINE_HOST_FUNCTION(functionConcatTypedArrays, (JSGlobalObject * globalObje
 
     auto arrayValue = callFrame->uncheckedArgument(0);
 
-    return flattenArrayOfBuffersIntoArrayBuffer(globalObject, arrayValue);
+    size_t maxLength = SIZE_MAX;
```

Comment:
`std::numeric_limits<size_t>` is more idiomatic

```suggestion
    size_t maxLength = std::numeric_limits<size_t>::max();
```

Reply:
Also, this should actually be `uint32_t` because the maximum size of a typed array is max uint32.